content/docs/concepts/options/transformations.md

```diff
@@ -17,6 +17,12 @@ The `transformations` resource option provides a list of transformations to appl
 
```

Comment:
I would suggest clarifying that the behavior of `transformations` on some component resources is that the children are simply not processed, contrary to the above paragraph. Is there a tip we could provide as to how to identify whether a given component transforms their children when using `transformations`?